session_1/hello_world/Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
```

Comment:
Please don't upload session 1 again

Reply:
I am sorry. Will not do it again

Reply:
Can you remove it from this MR too?

Reply:
could you please explain what you mean by MR?